test/helpers/index.js

```diff
@@ -75,7 +76,7 @@ function makePairs (n, multiplexMode = 'single') {
 }
 
 async function pipeStreamPairs (streams, messageSize, limit) {
-  const msg = Buffer.alloc(messageSize).fill('a')
+  const msg = b4a.alloc(messageSize).fill('a')
```

Comment:
fill is a bufferism, alloc(n, 'a') ?

Reply:
True! `TypedArray.fill()` only accepts numbers, not strings.